mlflow/store/tracking/rest_store.py

```diff
@@ -250,10 +250,27 @@ def get_metric_history(self, run_id, metric_key):
         :return: A list of :py:class:`mlflow.entities.Metric` entities if logged, else empty list
```

Comment:
Can we update the return type here and also return a namedtuple rather than a tuple?

Reply:
PagedList, right?